README.md

```diff
@@ -1,100 +1,137 @@
+2) [Knowledge processing machine](https://github.com/ostis-ai/sc-machine) with semantic network storage and agent-based knowledge processing
+3) [Web-oriented semantic interface](https://github.com/ostis-ai/sc-web) for users to interact with the intelligent system.
 
-# Installation
-## Using Docker
-You will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+To learn more about the platform, check out our [documentation](https://raw.githubusercontent.com/ostis-ai/ostis-web-platform/develop/docs/main.pdf).
 
-```sh
-git clone https://github.com/ostis-ai/ostis-web-platform
-cd ostis-web-platform
-docker compose pull
-```
+## Installation
 
-<details> 
-<summary> Building images locally </summary>
++ Quick start using Docker Compose
 
-e.g. when you want to launch a custom branch of sc-machine/sc-web
-#### Requirements
-1. **If you're using Windows**, please make sure you are using UNIX line endings inside the repository and `longpaths` are enabled:
+   Requirements: you will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+
+   ```sh
+   git clone https://github.com/ostis-ai/ostis-web-platform
+   cd ostis-web-platform
+   # download images from Docker Hub
+   docker compose pull
+   # build knowledge base
+   docker compose run machine build
+   # launch web platform stack
+   docker compose up
    ```
-   git config --local core.autocrlf true
-   git config --local core.longpaths true
-   git add --normalize .
+
+   <details>
+   <summary> Building docker images locally </summary>
+
+   This may come in handy e.g. when you want to use a custom branch of sc-machine or sc-web.
+
+   ### Requirements
+
+   1. **If you're using Windows**, please make sure you are using UNIX line endings inside the repository and `longpaths` are enabled:
+
+      ```
+      git config --local core.autocrlf true
+      git config --local core.longpaths true
+      git add --normalize .
+      ```
+
+   2. Enable Docker BuildKit. You can use `DOCKER_BUILDKIT=1` shell variable for this.
+
+   ### Build process
+
+   ```sh
+   git clone https://github.com/ostis-ai/ostis-web-platform
+   git submodule update --init --recursive
+   cd scripts
+   ./prepare.sh no_build_sc_machine no_build_sc_web # download all submodules without compilation.
+   cd ..
+   docker compose build
    ```
-2. Enable Docker BuildKit. You can use `DOCKER_BUILDKIT=1` shell variable for this.
 
-```sh
-git clone https://github.com/ostis-ai/ostis-web-platform
-git submodule update --init --recursive
-cd scripts
-./prepare.sh no_build_sc_web no_build_sc_web no_build_kb # download all submodules (without compilation)
-cd ..
-docker compose build
-```
-</details>
+   </details>
 
++ Natively
 
+   Note: Currently only Ubuntu is supported by this installation method. If you're going to use it, it might take a while to download dependencies and compile the components. Use it only if you know what you're doing!
```

Comment:
```suggestion
   Note: Currently, only Ubuntu is supported by this installation method. If you're going to use it, it might take a while to download dependencies and compile the components. Use it only if you know what you're doing!
```